python/ray/ml/preprocessors/imputer.py

```diff
@@ -82,16 +82,17 @@ def _get_most_frequent_values(
 ) -> Dict[str, Union[str, Number]]:
     columns = list(columns)
 
-    def get_pd_value_counts(df: pd.DataFrame) -> List[Counter]:
-        return [Counter(df[col].value_counts().to_dict()) for col in columns]
+    def get_pd_value_counts(df: pd.DataFrame) -> List[Dict[str, Counter]]:
+        return [{col: Counter(df[col].value_counts().to_dict()) for col in columns}]
```

Comment:
Does this still need to be wrapped in an array? If so do we need to unwrap it in line 92?